tests/mail_test.sh

```diff
@@ -220,6 +228,23 @@ function test_mail_parser()
   assert_equals_helper 'Set smtp pass' "$LINENO" "${options_values['sendemail.smtppass']}" "$expected"
 }
 
+function test_mail_send()
+{
+  local expected
+  local output
+  local ret
+
+  output=$(mail_send 'TEST_MODE')
+  expected="git send-email @^"
+  assert_equals_helper 'Testing send without options' "$LINENO" "$output" "$expected"
+
+  parse_mail_options '--to=mail@test.com'
+
+  output=$(mail_send 'TEST_MODE')
+  expected="git send-email --to=mail@test.com @^"
+  assert_equals_helper 'Testing send with to option' "$LINENO" "$output" "$expected"
```

Comment:
Can we have more tests but with the `dry` feature?

Reply:
I didn't implement this feature, but I could add it to the parser.

Reply:
Maybe you can convert the `TEST_MODE` to something that adds the flag `--dry-run`. I think it would be better to run the command instead of checking if it matches the expected string...